src/renderer/utils/Directories.ts

```diff
@@ -145,7 +145,7 @@ export class Directories {
 
         try {
             const symlinkPath = fs.readlinkSync(targetDir);
-            if (symlinkPath && fs.existsSync(path.join(symlinkPath, '/../.git'))) {
+            if (symlinkPath && fs.existsSync(path.join(symlinkPath, '/../../../.git'))) {
```

Comment:
maybe add a // TODO to change the way this check is done? This will break every time the depth of the directory is changed.

Reply:
I mean, we could crawl for the file through all possible paths, but that would also make the check a lot more resource heavy and time consuming. Idk if it's worth it given that this is the first time in FBW history that we changed the depth.